package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Todolist-App",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] Let's add basic commands for running Webpack in development mode, this helps to start your app locally.

Hint: 
```json
"start": "webpack serve --open",
```